src/libm-tester/tester3main.c

```diff
@@ -0,0 +1,27 @@
+//          Copyright Naoki Shibata 2010 - 2017.
```

Comment:
Why splitting this into main file and separate file? There is no I/O going on here as far as I can tell.

Reply:
Since tester3.c is compiled with options for enabling usage of vector extensions turned on, there is always possibility that the compiler generates instructions with that vector extension even without using intrinsics. In that case, the whole program crashes if that extension is not available on the computer. In our case, we want to make the testing succeed in that case, so the main function has to be separated and compiled without vector extension.

Reply:
It is already running on travis.

https://travis-ci.org/shibatch/sleef/jobs/317591746